doc/design/s3-backup-nese.md

```diff
@@ -0,0 +1,82 @@
+
+## Requirements
+
+### Functional requirements
+
+- **Replication.** Maintain a copy of the current contents of the primary bucket in the backup bucket.
+- **Backup.** Replicate all primary bucket writes and deletes to the backup bucket, maintaining a tape archive backup.
+
+### Non-functional requirements
+
+- **Cost control.** Enable tunable controls to help keep costs reasonable for the expected data volume.
+
+## Proposed Solution
+
+The solution leverages the widely used [NESE Tape](https://nese.readthedocs.io/en/latest/user-docs.html#nese-tape) service provided through the [Dartmouth Discovery HPC](https://rc.dartmouth.edu/hpc/discovery-overview/):
+
+NESE Tape provides higher density, lower cost storage, currently accessible via Globus. NESE Tape is composed of a tape system with several storage frames and 34 tape drives supporting up to 70 PB today with space available for expansion as needed.
+
+Each NESE Tape allocation comes with a disk-based staging area that is available via Globus. Users write to the staging area and then the data is migrated to tape based on a storage lifecycle policy. The default quota on this area is 10 TB or 2% of tape capacity, whichever is larger. There is also a minimum temporary hard quota set to 4 x staging-area space to allow for short term movement of larger amounts of data.
+
+### How It Works
+
+Unlike the [Deep Glacier approach](https://github.com/dandi/dandi-archive/blob/b3e0a9df4188533723fb2ad4a95506aa724fc089/doc/design/s3-backup.md), some management of file transfers may be necessary, either performed by our team or by the HPV IT team, based on the NESE file transfer ruleset:
+
+- all newly written files > 2 hr of modification time are copied to tape.
+- if fileset quota > 99%; files are stubbed (replaced with a small pointer) down to 75% quota.
+- files with access time age > 2 weeks are stubbed.
+- files < 100 MB copied to tape, but also remain on disk.
+
+## Limitations and Considerations
+
+- **Upper bound on total storage**: No quota for an upper limit of long-term storage has yet been provided, aside from the grand 70 PB total of the entire NESE store; the providers claim expansion is possible as need arises.
+- **Replication is eventually consistent**: No guarantees about replication speed (the time between an object finishing upload into the primary bucket and when it is available in the backup bucket) are provided. Using previous bandwidth experience to the old Dropbox backup, multi-gigabit speeds should be possible and is expected to keep up with ingest rates on the primary S3 bucket.
+
+## Cost
+
+Note that unlike the [Deep Glacier approach](https://github.com/dandi/dandi-archive/blob/b3e0a9df4188533723fb2ad4a95506aa724fc089/doc/design/s3-backup.md), no egress cost would be induced from the data transfer as all operations are strictly under the open data bucket.
```

Comment:
Do we know this for sure? AWS would probably take notice if our data transfer costs suddenly started spiking on a recurring basis due to a backup script downloading the entire multi-PB bucket.
As far as I understand, this isn't a _technical_ limitation of the open data account; from our perspective, the billing for the entire AWS account is paid for by the AWS Open Data program. 

So, we could apply the same thinking to the Deep Glacier approach - if we put the backup bucket in the open data AWS account, the storage costs would also theoretically be covered by the open data program. In practice, it's more of a policy issue - is AWS willing to pay for backup? Framed that way, I don't think there is a clear winner between the two approaches in terms of AWS transfer costs.